js/constants/constants.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
     ERROR_SCREEN: "errorScreen"
   },
 
+  CUSTOM_EVENTS: {
```

Comment:
I'm cool with the custom events.  @aeng7 Which var name do think is better INITIAL_PLAY_REQUESTED or DEFERED_INITIAL_PLAY?